doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+## Mattermost Config Settings
+___
+Config settings allow system administrators to customize their teams Mattermost experience. Warning: system administrators should be cautious to only grant admin privileges to team members who understand the critical nature of the settings they are adjusting.
+
+
+## System Console Settings:
+These settings can be modified from the System Console user interface or config.json.
```

Comment:
Maybe add a quick blurb on where to find the button to switch to the system console? Let's be super obvious is we can